src/faucet.e2e.ts

```diff
@@ -150,4 +160,24 @@ describe("Faucet E2E", () => {
       "balance did not increase.",
     );
   });
+
+  test("The web endpoint fails on wrong parachain", async () => {
+    const userAddress = randomAddress();
+
+    try {
+      await webEndpoint.post("/drip/web", { address: userAddress, recaptcha: "anything goes", parachain_id: "100" });
+
+      // should not have succeed
+      throw new Error("Unexpected success");
+    } catch (error) {
+      if (axios.isAxiosError(error)) {
+        const axiosError = error as AxiosError<{ error: string }>;
+        expect(axiosError.response?.data.error).toEqual(
+          "Parachain invalid. Be sure to set a value between 1000 and 9999",
+        );
+      } else {
+        throw error;
+      }
+    }
```

Comment:
Wouldn't it work somewhat like this?

```ts
const promise = webEndpoint.post("/drip/web", { address: userAddress, recaptcha: "anything goes", parachain_id: "100" });

await expect(promise).rejects.toThrow(AxiosError);
await expect(promise).rejects.toMatchObject({response: {data: {
  error: "Parachain invalid. Be sure to set a value between 1000 and 9999"
}}});
```

Reply:
It would! Thank you for that!

Reply:
`throw new Error("Unexpected success");` lmao